test/ncengine/serialize/ComponentSerialization_integration_tests.cpp

```diff
@@ -155,6 +156,15 @@ TEST(ComponentSerializationTests, RoundTrip_particleEmitter_preservesValues)
     EXPECT_EQ(expectedInfo.kinematic.scaleOverTimeFactor, actualInfo.kinematic.scaleOverTimeFactor);
 }
 
+TEST(ComponentSerializationTests, RoundTrip_directionalLight_preservesValues)
+{
+    auto stream = std::stringstream{};
+    const auto expected = nc::graphics::DirectionalLight{nc::Vector3::Splat(1.0f)};
```

Comment:
The serialization is so trivial here that it won't matter, but I prefer making the values of 'expected's here different from their defaults to be super sure we're reading data correctly (typical deserialization workflow default constructs before overwrite).